### DISCUSSION TITLE
RustDesk Autostart on IoT Device

### DISCUSSION THREAD

**Discussion by Orholt** [rustdesk/rustdesk#3331] [General]
Hi, I'm trying to integrate RustDesk with IoT device (Android Tablet running Android 12 with custom Pixel ROM)
I own system signing key in order to sign .apk (if needed) so I can get all permissions if needed.
To be honest, I expect that to work like TeamViewer Host or anything like that (Install & Run)

My question is as flowing:
Can the application start automatically with the start of the machine and not require manual granting of permissions?

It will be a huge relief due to the fact that this functionality is now essential for me.

Thanks for answers

**Comment by zclikezc** [comment]
Hi I tried to use Rustdesk on my android tablet, but it seems if I try to use pc to control the tablet, that won't work, with "connection refused" error, may I ask how you solved this problem? thank you

**Reply by Orholt** [reply]
To be honest, In my case everything works fine, even on nightly. Maybe try to set up own relay server? Or at least check your network configuration.

**Reply by zclikezc** [reply]
Thx, may I ask how to check the network config on Android?
